sql/flow.go

```diff
@@ -141,34 +142,57 @@ func CommonDockerUtil(cmd, args []string, flags map[string]string, mountDirs []s
 		"",
 	)
 	if err != nil {
-		return fmt.Errorf("docker container creation failed %w", err)
+		return "", fmt.Errorf("docker container creation failed %w", err)
 	}
 
 	if err := cli.ContainerStart(ctx, resp.ID, types.ContainerStartOptions{}); err != nil {
-		return fmt.Errorf("docker container start failed %w", err)
+		return "", fmt.Errorf("docker container start failed %w", err)
 	}
 
 	statusCh, errCh := cli.ContainerWait(ctx, resp.ID, container.WaitConditionNotRunning)
 	select {
 	case err := <-errCh:
 		if err != nil {
-			return fmt.Errorf("docker container wait failed %w", err)
+			return "", fmt.Errorf("docker container wait failed %w", err)
 		}
 	case <-statusCh:
 	}
 
 	cout, err := cli.ContainerLogs(ctx, resp.ID, types.ContainerLogsOptions{ShowStdout: true, ShowStderr: true})
 	if err != nil {
-		return fmt.Errorf("docker container logs fetching failed %w", err)
+		return "", fmt.Errorf("docker container logs fetching failed %w", err)
 	}
 
-	if _, err := IoCopy(os.Stdout, cout); err != nil {
-		return fmt.Errorf("docker logs forwarding failed %w", err)
+	outputLogs := ""
+	if returnStdout {
+		rescueStdout := os.Stdout
```

Comment:
Could this be separated into a helper function?

Reply:
moved it to a function.

Reply:
we no longer use this.